indico/modules/events/surveys/templates/survey_announcement.html

```diff
@@ -0,0 +1,28 @@
+{% set survey_count = surveys|length %}
+
+<div class="infoline surveys">
+    <span class="icon icon-clipboard"></span>
+    <div class="text">
+    <div class="label">
+        {% trans %}
+            Open surveys
```

Comment:
Just Survey/Surveys might be better. We don't show the announcement if there isn't an open one anyway